### DISCUSSION TITLE
Go to Definition

### DISCUSSION THREAD

**Discussion by StevenXL** [LazyVim/LazyVim#1665] [Q&A]
Hi! 

I am trying to use the `go to definition` functionality by using the `gd` keys, but it is not working. I don't know what other information I can / should provide. Is there a log somewhere that might be able to point me to what is wrong? 

This is in a TypeScript file, and other features such as type information when I press `K` works, `gr` opens up telescope with some stuff pre-filled, etc. etc.

**Comment by fredrikaverpil** [comment]
What do you see when you run `:LspInfo`, when in the `.ts` file?

**Reply by StevenXL** [reply]
User error on my part. It turns out that I was using `gd` on the variable name, not the function call. In other words: `const someVar = someFunc(....)`, I was on `someVar`. 🤦🏽. 

Thank you for your response!
